internal/factsengine/gatherers/sbd.go

```diff
@@ -72,22 +70,21 @@ func (g *SBDGatherer) Gather(factsRequests []entities.FactRequest) ([]entities.F
 	return facts, nil
 }
 
-func loadSbdConf(sbdConfigPath string) (map[string]string, error) {
-	sbdConfigFile, err := os.Open(sbdConfigPath)
-	if err != nil {
-		return nil, SBDConfigFileError.Wrap(err.Error())
-	}
+func loadSbdConfig(sbdConfigPath string) (map[string]string, error) {
```

Comment:
I find this function a bit useless here.
Why don't we simply use `cluster.LoadSbdConfig` up in the code, and a put `SBDConfigDecodingError` together with the error message.
The result would be the same, and we would have less "boilerplate" code